environments/se/core/src/main/java/org/jboss/weld/environment/se/WeldContainer.java

```diff
@@ -209,6 +210,9 @@ private void complete() {
         this.creationalContext = beanManager().createCreationalContext(null);
         this.instance = beanManager().getInstance(creationalContext);
         this.event = beanManager().event();
+    }
+
+    private void fireContainerInitializedEvent() {
         beanManager().fireEvent(new ContainerInitialized(id), Initialized.Literal.APPLICATION);
         WeldSELogger.LOG.weldContainerInitialized(id);
```

Comment:
I didn't move it. The logging [was **after**](https://github.com/weld/core/blob/master/environments/se/core/src/main/java/org/jboss/weld/environment/se/WeldContainer.java#L213)  the event.
But you are right, that it makes sense to swap those lines.